frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-details-card.tsx

```diff
@@ -15,24 +15,23 @@ import {
   resourcePath,
 } from '@console/internal/components/utils';
 import { VMDashboardContext } from '../../vms/vm-dashboard-context';
-import { isVMRunning } from '../../../selectors/vm';
-import { getVMStatus } from '../../../statuses/vm/vm';
 import { VirtualMachineModel } from '../../../models';
-import { getVmiIpAddressesString } from '../../ip-addresses';
+import { getVmiIpAddresses } from '../../../selectors/vmi/ip-address';
 import { VM_DETAIL_OVERVIEW_HREF } from '../../../constants';
+import { findVMPod } from '../../../selectors/vm/combined';
 
 export const VMDetailsCard: React.FC<VMDetailsCardProps> = () => {
   const vmDashboardContext = React.useContext(VMDashboardContext);
-  const { vm, vmi, pods, migrations } = vmDashboardContext;
+  const { vm, vmi, pods } = vmDashboardContext;
+  const vmLike = vm || vmi;
 
-  const vmStatus = getVMStatus({ vm, vmi, pods, migrations });
-  const { launcherPod } = vmStatus;
+  const launcherPod = findVMPod(vmLike, pods);
 
-  const ipAddrs = getVmiIpAddressesString(vmi, vmStatus);
+  const ipAddrs = getVmiIpAddresses(vmi).join(', ');
```

Comment:
getVmiIpAddressesString can be removed now

Reply:
done